dev/src/webpack/getDefaultWebpackConfig.ts

```diff
@@ -45,11 +45,78 @@ export function getDefaultWebpackConfig(options: DefaultWebpackOptions): webpack
         logCore(loggingVerbosity, messageVerbosity, ...args);
     }
 
+    const plugins = [
+        // Copy files to dist folder where the runtime can find them
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+        new CopyWebpackPlugin({
+            patterns: [
+                // Test files -> dist/test (these files are ignored during packaging)
+                {
+                    from: path.posix.join(options.projectRoot.replace(/\\/g, '/'), 'out', 'test', '*.*'),
+                    to: path.posix.join(options.projectRoot.replace(/\\/g, '/'), 'dist', 'test'),
+                    noErrorOnMissing: true
+                },
+                {
+                    from: path.join(options.projectRoot, 'node_modules', '@microsoft', 'vscode-azext-azureutils', 'resources', '**', '*.svg'),
```

Comment:
Removed some of the old copy paths because they were old azuretools packages that we weren't using anymore.